BetaCameras/ifm/O3D3xx.cs

```diff
@@ -42,7 +42,7 @@ public bool Resolution100k
             {
                 if (!IsConnected)
                 {
-                    return false;
+                    throw new NotConnectedException($"{nameof(O3D3xx)}: {nameof(Resolution100k)} connect be read while disconnected.");
```

Comment:
I guess there is typo in this message: "connect" -> "cannot" (it occurs at multiple positions)